Source/Task/ThreadPool_win32.cpp

```diff
@@ -61,10 +61,21 @@ class ThreadPoolImpl
         }
     }
 
-    void Submit() noexcept
+    // SubmitThreadpoolWork (seems to be an alias for NTDLL.TpPostWork) cannot be called during application shutdown
+    // It throws: Exception thrown at 0x00007FFD726AF220 (ntdll.dll) in CTA_PC.exe: 0xC000000D: An invalid parameter was passed to a service or function. occurred
```

Comment:
Should remove the name of the exe from this comment. 
Also, I think libhttpclient repo maintains a 120 or 140 char limit and this line looks like its over 160.